.github/workflows/continuous-integration.yml

```diff
@@ -29,6 +30,7 @@ jobs:
 
   test-integration:
     name: Integration Tests
+    timeout-minutes: 120
```

Comment:
Two hours!? 😨

Reply:
Yeah because certain changes that modify all builds will run each builder test sequentially. I've seen it go over an hour.